OpenRA.Game/Settings.cs

```diff
@@ -142,6 +142,7 @@ public class SoundSettings
 
 		public bool CashTicks = true;
 		public bool Mute = false;
+		public bool DisplayAltTexts = false;
```

Comment:
The original games did audio + visual notifications, although those notifications did appear somewhere other than the chat.  There are discussions from many years ago (long before #12954) about us implementing these, but nobody picked up the feature until now.